torch/_dynamo/decorators.py

```diff
@@ -236,6 +236,20 @@ def mark_static(t, index=None):
             mark_static(t, i)
 
 
+@forbid_in_graph
+def mark_static_input(t, guard=True):
```

Comment:
We could also do mark_static_address ?

Reply:
yeah I thought this name for a while, address is better, ty!